### ISSUE TITLE
Opening DWG in Alphacam

### ISSUE THREAD

**Issue by emdant13** [maxvfischer/DIY-arcade#1]
Hi Max! 

I'm so excited to try your project! Would you be able to post another DWG with your parts as 2d geometries instead of a 3d model? The cabinet in the current DWG also doesn't import to the correct size, would that be a problem on my end? Thank you! :)

**Comment by maxvfischer** [comment]
Hi @emdant13!
I'm glad that the tutorial might be of help :) Good luck and please share images here when your done! 

That's weird, when I open the .dwg-file in Fusion 360, it has the correct size. What CAD-program are you using?

Regarding 2d files, will .dxf files work for you? Exporting 2d .dxf-files in Fusion 360 is very simple, but I'm not sure about .dwg. If you really need .dwg I can investigate it further.

**Comment by emdant13** [comment]
I use Alphacam and I can use .dwg and .dxf. Now I'm thinking it didn't convert your measurements properly. Did you use millimeters? This would explain why it comes in at a couple thousand inches tall for me. XD Thanks for your help!

**Comment by maxvfischer** [comment]
Oh, yeah it's in millimeters. Thanks for the feedback, I'll update the README and make that clear :) 

Great, then I'll upload the 2d-files in .dxf, hopefully later today.

**Comment by maxvfischer** [comment]
@emdant13 I've pushed all the panels as 2d .dxf-files to master (see PR  #2).

Good luck with the build! Please share some images when you're done and feel free to ask questions :)

NOTE: I've only CNC-cut the `side_panel`, the `screen_panel` and the `button_panel`, the rest of the panels were manually cut out.

**Comment by maxvfischer** [comment]
@emdant13 Any update on your build? :)

**Comment by emdant13** [comment]
Hello again!
My build is finally finished and with its new owner. Thank you so much for sharing your design! I did tweak it a little to fit my needs and it came out great! I added dados to make assembly simpler and three drawers under the controls for storage. I also attached the control panel with a hinge and a door on the back of the body for easy access to plugs, wires, etc. Everything runs on Retropie and the friends I gifted it to are so excited! 
![20210814_150653](https://user-images.githubusercontent.com/78153376/129463822-6938a65b-e06a-407a-b8e0-53144c5f28a0.jpg)
